internal/armada/server/submit_from_log.go

```diff
@@ -510,7 +506,11 @@ func (srv *SubmitFromLog) CancelJobSet(ctx context.Context, queueName string, jo
 	} else if err != nil {
 		return true, err
 	}
+	//TODO Set userid correctly so we know who actually cancelled the jobset
```

Comment:
When I see todo in code, I think two things:

should you do this now while the code is fresh?

or should you file this as a issue so we have it in our backlog?

Reply:
I didn't add this, I spotted this bug while doing this PR.

We should probably raise an issue, as there is no obvious solve